source/simulator/assembly.cc

```diff
@@ -1712,7 +1712,7 @@ namespace aspect
                 += (
                      (time_step * (conductivity + nu)
                       * (scratch.grad_phi_field[i] * scratch.grad_phi_field[j]))
-                     + ((time_step * (current_u * scratch.grad_phi_field[j] * scratch.phi_field[i]))
+                     + ((time_step * (current_u * scratch.grad_phi_field[i] * scratch.phi_field[j]))
```

Comment:
Haha, of course.  For some reason I thought they were both grad_phi_field last night.